### DISCUSSION TITLE
Simple DI - Container

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#33] [General]
# Simple DI - Container

This blog post will show you a very simple Dependency Injection container.
This will give a better understanding of what Dependency Injection is and how it is done. And sure we will see how this is related to IoC - Inversion of Control.

https://steven-giesel.com/blogPost/8cd29874-1bb5-4df5-8e0a-293a293eb25a

**Comment by EngincanV** [comment]
Great article! Thanks Steven.

**Reply by linkdotnet** [reply]
Thanks @EngincanV for the supportive words! I did enjoy making it :D
